composer.json

```diff
@@ -12,12 +12,12 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "nikic/php-parser": "~0.9.5",
+        "nikic/php-parser": "~1.3.0",
```

Comment:
You should really update all the old class names otherwise upgrading to v2 will be even harder as the aliases are no longer registered.